Automattic/newspack-plugin
# feat(newspack-ui): update newspack ui to accomodate modal checkout block

### All Submissions:






### Changes proposed in this Pull Request:



Part of `1206197591054328/1206405274735557/f`

This adds a spinner to newspack ui.

### How to test the changes in this Pull Request:

See https://github.com/Automattic/newspack-blocks/pull/1649 and https://github.com/Automattic/newspack-blocks/pull/1653 for testing steps

### Other information:
